code/modules/cargo/supplypod.dm

```diff
@@ -384,7 +384,9 @@
 			return FALSE
 		if(!obj_to_insert.anchored && reverseOptionList["Unanchored"])
 			return TRUE
-		if(obj_to_insert.anchored && reverseOptionList["Anchored"])
+		if(obj_to_insert.anchored && !ismecha(obj_to_insert) && reverseOptionList["Anchored"]) //Mecha are anchored but there is a separate option for them
+			return TRUE
+		if(ismecha(obj_to_insert) && reverseOptionList["Mecha"])
 			return TRUE
```

Comment:
If you move the mecha checks to before the anchored checks, you won't need the check for mecha in the anchored checks.

Reply:
I'm not sure that's true. You'd still get mechs if you selected anchored but not mechs. Which I guess is technically true because mechs are anchored but it's not a distinction I'd expect everyone using these to know.

Reply:
I guess. I can think of at least one other implementation such as checking reverseOptionList("Mecha"]) first then returning TRUE or FALSE depending on the ismecha() check afterwards, but I guess that's just arguing semantics and it gets the job done either way.

Also, on the topic of reverseOptionList, can you do a find-and-replace to rename it to something a bit snake_case-ier for me?

Reply:
Done.